components/authn-service/tokens/pg/testconstants/testconstants.go

```diff
@@ -1,27 +0,0 @@
-package testconstants
-
-// SkipPGMessageFmt is the friendly message the developer sees when any tests
-// requiring postgres are skipped because the connection failed.
-//
-// Yes those lines are long, but if we linebreak, the output ends up starting
-// with TAB characters, which makes it harder to copy-paste the commands.
-const SkipPGMessageFmt = `
```

Comment:
Like @blakestier was saying, this is a much more actionable error message. Do we need to remove it?

Reply:
don't think that's what we want people to do? this made sense when we weren't automating with the makefile.